diagnostic.md

```diff
@@ -92,7 +115,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+A font family of cursive and a color of red.
```

Comment:
is this because the class that makes the class that makes the color yellow occurs later in the html?